android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/responses/ShowConstantTbtResponseTests.java

```diff
@@ -3,7 +3,7 @@
 import com.smartdevicelink.marshal.JsonRPCMarshaller;
 import com.smartdevicelink.protocol.enums.FunctionID;
 import com.smartdevicelink.proxy.RPCMessage;
```

Comment:
@kostyaBoss I would revert these changes in `ShowConstantTbtResponseTests.java`.  This PR only deals with the `ShowConstantTbtRequest` so these changes are out of scope of the PR.